lib/LANraragi/Model/Config.pm

```diff
@@ -40,8 +40,8 @@ sub get_redis {
         reconnect => 3
     );
 
-    if ($ENV{BREWMODE}) {
-        $redis->config_set("dir", $ENV{HOME} . "/Library/Application Support/LANraragi/");
+    if ($ENV{LANDIR}) {
+        $redis->config_set("dir", $ENV{LANDIR});
```

Comment:
Since you're using the redis configuration file, this line isn't necessary anymore.